ern-local-cli/src/lib/publication.js

```diff
@@ -142,7 +145,7 @@ version: string, {
   try {
     const plugins = await cauldron.getNativeDependencies(napDescriptor)
     const miniapps = await cauldron.getContainerMiniApps(napDescriptor)
-    const pathToYarnLock = await cauldron.getPathToYarnLock(napDescriptor)
+    const pathToYarnLock = await cauldron.getPathToYarnLock(napDescriptor, 'container')
```

Comment:
would be it be nicer to export `container` as a constant in a way it is accessible across the `ern-cli`

Reply:
agreed. that said, it is used in different `ern` modules, so I'll expose this constant from `ern-core`